README.md

```diff
@@ -1,29 +1,39 @@
-# Kubernetes Template Project
+# Maestro
 
-The Kubernetes Template Project is a template for starting new projects in the GitHub organizations owned by Kubernetes. All Kubernetes projects, at minimum, must have the following files:
+Maestro is a Kubernetes operator built for deploying software with interesting lifecycles.
 
-- a `README.md` outlining the project goals, sponsoring sig, and community contact information
-- an `OWNERS` with the project leads listed as approvers ([docs on `OWNERS` files][owners])
-- a `CONTRIBUTING.md` outlining how to contribute to the project
-- an unmodified copy of `code-of-conduct.md` from this repo, which outlines community behavior and the consequences of breaking the code
-- a `LICENSE` which must be Apache 2.0 for code projects, or [Creative Commons 4.0] for documentation repositories, without any custom content
-- a `SECURITY_CONTACTS` with the contact points for the Product Security Team 
-  to reach out to for triaging and handling of incoming issues. They must agree to abide by the
-  [Embargo Policy](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#embargo-policy) 
-  and will be removed and replaced if they violate that agreement.
+## Motivation
 
-## Community, discussion, contribution, and support
+- Inspired by DC/OS Commons
+- Need for an abstraction framework around complex software lifecycles
+- ???
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+## Installation Instructions
```

Comment:
This doesn't seem like an installation onto a cluster, i think the `make deploy` command captures that.  Looking at the `deploy` Makefile argument we might want to explicity keep the output of deploy:
```
	kustomize build config/default \
```
as a file, and be able to have an install guide reference that file hosted on Github directly.